test/ecto/changeset_test.exs

```diff
@@ -724,4 +725,21 @@ defmodule Ecto.ChangesetTest do
                 |> validate_number(:upvotes, greater_than: 100, less_than: 0, message: "yada")
     assert changeset.errors == [upvotes: {"yada", 100}]
   end
+
+  test "validate_confimation/3" do
```

Comment:
Alrighty then. I'll get working these requests today. @crismali I'll turn on spell check this time lol :p